LBH.AdultSocialCare.Api/Helpers/CurrentDateTimeOffsetProvider.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+
+namespace LBH.AdultSocialCare.Api.Helpers
+{
+    public class CurrentDateTimeOffsetProvider : ICurrentDateTimeOffsetProvider
+    {
+        public DateTimeOffset Now => DateTimeOffset.Now;
+        public DateTimeOffset UtcNow => DateTimeOffset.UtcNow;
+        public static CurrentDateTimeOffsetProvider DateTimeOffset => new CurrentDateTimeOffsetProvider();
```

Comment:
Why we need this static property CurrentDateTimeOffsetProvider.DateTimeOffset ?

Typical usage is:

- Wherever we need DateTime.Now we create a property like DateTimeProvider, set it to our default implementation (CurrentDateTimeOffsetProvider ) and use this DateTimeProvider.Now instead of DateTime.Now
- For test we set this property to a mock which returns any given value for Now property
- So in CareChargesGateway we should use DateTimeProvider.Now instead of DateTime.Now to make it testable
- static elements should be avoided wherever possible, as they complicates testing

Reply:
It would allow doing something like `CurrentDateTimeOffsetProvider.DateTimeOffset.Now` or `CurrentDateTimeOffsetProvider.DateTimeOffset.UtcNow` if you inherit this class directly. I am just going to remove it anyway because it has not been used yet

Reply:
And done